<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget{
border-top-left-radius:150px;
border-top-right-radius:150px;
}</string>
  </property>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>1238</x>
     <y>7</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#closeBtn{
background-color:transparent;
border-image: url(:/resources/Close.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="minimizeBtn">
   <property name="geometry">
    <rect>
     <x>1203</x>
     <y>7</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#minimizeBtn{
background-color:transparent;
border-image: url(:/resources/Minimize.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="titleBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>45</y>
     <width>1280</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#titleBar{
background-color:transparent;
border-image:url(:/resources/Div Line.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="pomodoroBuddy">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>232</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pomodoroBuddy{
background-color:transparent;
border-image: url(:/resources/pomodorobuddy.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopBtn">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>329</y>
     <width>163</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stopBtn{
background-color:transparent;
border-image:url(:/resources/STOP.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>559</x>
     <y>329</y>
     <width>163</width>
     <height>96</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#startBtn{
background-color:transparent;
border-image: url(:/resources/START.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="interruptBtn">
   <property name="geometry">
    <rect>
     <x>957</x>
     <y>329</y>
     <width>163</width>
     <height>96</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#interruptBtn{
background-color:transparent;
border-image:url(:/resources/INTERRUPT.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="timer">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>68</y>
     <width>406</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#timer{
background-color:transparent;
border-image: url(:/resources/counter.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="statsBtn">
   <property name="geometry">
    <rect>
     <x>844</x>
     <y>458</y>
     <width>389</width>
     <height>114</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#statsBtn{
background-color:transparent;
border-image:url(:/resources/STATS.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsBtn">
   <property name="geometry">
    <rect>
     <x>844</x>
     <y>573</y>
     <width>389</width>
     <height>114</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#settingsBtn{
background-color:transparent;
border-image: url(:/resources/SETTINGS.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="counter">
   <property name="geometry">
    <rect>
     <x>446</x>
     <y>458</y>
     <width>389</width>
     <height>229</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#counter{
background-color:transparent;
border-image:url(:/resources/POMODORO COUNTER.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="highscore">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>458</y>
     <width>389</width>
     <height>229</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#highscore{
background-color:transparent;
border-image:url(:/resources/POMODORO HIGHSCORE.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="highscoreText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>381</width>
     <height>133</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>100</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(112, 112, 112);</string>
   </property>
   <property name="text">
    <string>999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pomCounter">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>510</y>
     <width>381</width>
     <height>133</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>100</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(112, 112, 112);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="timerCounter">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>401</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>109</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(112, 112, 112);</string>
   </property>
   <property name="text">
    <string>25:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="border1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#border1{
background-color:transparent;
border-image:url(:/resources/Div Line.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="border2">
   <property name="geometry">
    <rect>
     <x>1279</x>
     <y>0</y>
     <width>1</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#border2{
background-color:transparent;
border-image:url(:/resources/Div Line.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="border3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#border3{
background-color:transparent;
border-image:url(:/resources/Div Line.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="border4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>719</y>
     <width>1280</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#border4{
background-color:transparent;
border-image:url(:/resources/Div Line.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="pomodoroPenguin">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>68</y>
     <width>250</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pomodoroPenguin{
background-color:transparent;
border-image:url(:/resources/pomodoropenguin.png);
background: none;
border: none;
background-repeat: none;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
